examples/jsm/nodes/math/MathUtilsNode.js

```diff
@@ -0,0 +1,15 @@
+import { sub, mul, div, add } from './OperatorNode.js';
```

Comment:
I think we also can rename this file to just `MathUtils` and maybe even merge TriNoise3D in this one.